tests/e2e/e2e_test.go

```diff
@@ -3557,7 +3529,7 @@ func TestE2E(t *testing.T) {
 
 				// We need to check that the remediation is auto-applied and save
```

Comment:
We could check that the check we're remediating failed before we attempt to fix it. I'm thinking about if the defaults change or for some reason the check starts to pass automatically, we wouldn't be testing the remediation, would we?

Reply:
If the check pass, it will not generate a remediation here

Reply:
Don't we want to test that it does?

Reply:
I think it is being checked, see line #3523[https://github.com/ComplianceAsCode/compliance-operator/pull/197/files#diff-7d1fc1e232b55677582b51e2623aae6d8c15b5f0960ac6100fc7d733cb795dacR3523]

Reply:
do you mean check the result of the first scan:
```
err = waitForSuiteScansStatus(t, f, namespace, suiteName, compv1alpha1.PhaseDone, compv1alpha1.ResultNonCompliant)
```

Reply:
Aha - got it, thanks!

Reply:
the GitHub diff sometimes doesn't show the full code haha

Reply:
thanks for the review!